lib/Core/Persistence/FieldHandler/Custom/CountryFieldHandler.php

```diff
@@ -36,4 +45,9 @@ public function getLegacyValue(Value $value, FieldDefinition $fieldDefinition):
     {
         return FieldValue::withStringValue($fieldDefinition->id, implode(', ', array_column($value->countries, 'Alpha2')));
     }
+
+    public function fromLegacyValue(FieldValue $legacyData)
+    {
+        return $this->countryType->fromHash(explode(', ', $legacyData->getDataText()));
```

Comment:
Let's try to make this a little bit more robust; explode by `,` (without space) and apply `trim` to the elements of the resulting array.

Reply:
fixed in 316673c5 commit, also added return type in 34ebdf25